pyroomacoustics/room.py

```diff
@@ -702,11 +723,40 @@ def plot(self, img_order=None, freq=None, figsize=None, no_axis=False, mic_marke
 
             return fig, ax
 
-    def plot_rir(self, FD=False):
+    def plot_rir(self, pairs=None, FD=False):
```

Comment:
I'm not completely satisfied with keyword `pairs` since it does not only describe pairs, but also select a microphone. Maybe we could refactor it as `select` or something like that ?